source/reference/api.txt

```diff
@@ -9,6 +9,8 @@ API
 .. toctree::
    :titlesonly: 
 
+   /reference/api/commit
    /reference/api/pause
+   /reference/api/progress
```

Comment:
can we order these pages under API? 

-Start
-Progress
-Pause
-Resume
-Commit
-Reverse

Reply:
Hmm... are we sure that is the best approach? That is likely the order in which users will run these commands, but users looking for a specific endpoint may find it most useful for them to be alphabetized. If you feel strongly about your suggested order, I'm happy to make this edit.

Reply:
Ah, I missed the alphabetized ordering. I don't foresee us adding more APIs in the near future and 'commit' at the top of the list threw me off. Personally, I prefer the list based on the most frequently used commands.